packages/expo-media-library/src/MediaLibrary.ts

```diff
@@ -290,7 +292,13 @@ export type AssetRef = Asset | string;
 // @docsMissing
 export type AlbumRef = Album | string;
 
-export { PermissionStatus, PermissionExpiration, EXPermissionResponse, Subscription };
+export {
+  PermissionStatus,
+  PermissionExpiration,
+  EXPermissionResponse,
```

Comment:
I'm not sure if this is correct @tsapeta. I think we need to export the `PermissionResponse` here, I think the names could be improved but:

- Original `PermissionResponse` imported as `EXPermissionResponse`
- Added properties to `PermissionResponse`
    > some libraries name them `<package>PermissionResponse`, e.g. `MediaLibraryPermissionResponse`
- Export that modified permission response type here, in our case `PermissionResponse`

Reply:
Thanks to @Simek, this is working fine. We export the `PermissionResponse` separately, and this is just to allow a link like:

![Screenshot 2021-08-17 at 21 13 50](https://user-images.githubusercontent.com/1203991/129786595-6a62b5db-4c95-4162-909c-428166d4eb6f.png)

But it still might be better to rename the type to `MediaLibraryPermissionResponse`, just to avoid confusion.